src/resources/Location.php

```diff
@@ -11,6 +11,8 @@
  * @see Locations::get()
  * @see Airports::get()
  * @see https://developers.amadeus.com/self-service/category/air/api-doc/airport-and-city-search/api-reference
+ * @see RecommendedLocations::get()
+ * @see https://developers.amadeus.com/self-service/category/trip/api-doc/travel-recommendations/api-reference
```

Comment:
Add the docs of Airport Nearest Relevant API as well, and on line 10 Travel Recommendations API